src/sphinx_tags/__init__.py

```diff
@@ -28,7 +28,7 @@ class TagLinks(SphinxDirective):
 
     # Custom attributes
     separator = ","
-    intro_text = "Tags: "
+    intro_text = "In categories: "
```

Comment:
This could probably be configured as well, I guess. We can do this in a follow-up though!

Reply:
Right, makes sense!